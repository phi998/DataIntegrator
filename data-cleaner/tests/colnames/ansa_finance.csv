other,Article Content,Unknown,Unknown,Unknown,Title,Title,Category,Unknown,Title,Title,Price,Title,Category,Title,Article Content,Unknown,Category,Unknown,cookie_policy,Unknown,Unknown,Article Content,Unknown,Unknown,Title,Category,Title,Publication Date,Title,Unknown,Unknown,Publication Date,Publication Date,Title,Category,Title,Title,Unknown,other,Title,Category,Title,Title,Title,Category,Category,Category,Title,Category,Title,Other,Category,Title,Category,Title,Category,Category,Category,Title,Unknown,Category,Category,Title,Category,Category,Category,Category,Category,Category,Author,Title,Category,Title,Other,Unknown,Title,Category,Title,Title,other,Category,Category,Category,Title,Category,Title,Category,Category,Publication Date,Category,Unknown,Category,Category,Category,Category,Category,Title,Category,Title,Title,Article Content,Unknown,Unknown,Category,Unknown,Title,other,Category,Title,Category,Category,Title,Title,Unknown,Category,Category,Other,Category,Category,Title,Title,Category,Category,Title,Title,Category,Title,Category,other,Category,Category,Category,other,Category,Category,Publication Date,Unknown,Category,Title,Category,Title,Title,Category,Category,Title,Title,Unknown,Category,Category,Title,Unknown,Title,Article Content,Category,Publication Date,Title,Title,Category,Category,FAQ,Title,Title,Category,Title,Title,Title,Title,Category,Other,Category,Unknown,Title,Title,Article Content,Publication Date,Title,other,Title,Title,Unknown,Unknown,Author,Title,Title,Category,Category,Category,Category,Category,Article Content,Unknown,Title,Title,Category,Category,Title,Title,Category,Category,Category,Category,Category,Author,Title,Title,Unknown,Category,Publication Date,Title,Article Content,Title,Publication Date,Author,Publication Date,Unknown,Unknown,Article Content,Publication Date,Title,Unknown,Category,Unknown,Unknown,Title,Article Content,Article Content,Unknown,Utilizza,Title,Title,Title,Title,Publication Date,other,Unknown,Publication Date,Title,Unknown,other,Title,Unknown,Unknown,Title,Category,Title,Title,other,Unknown,Title,Title,Title,other,Category,Title,Title,Title,Title,Category,Unknown,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Article Content,Publication Date,Author,Title,Title
