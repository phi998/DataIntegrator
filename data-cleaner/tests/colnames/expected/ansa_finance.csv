.,Title,.,.,.,.,.,.,.,.,.,Price,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Publication Date,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Article Content,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Article Content,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Article Content,Location,Publication Date,Author,.,.,.,Article Content,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Article Content,Article Content,.,.,.,.,.,.,.,.,.,.,.,.,.,.
